### PR TITLE
correct threshold used

### PR DESCRIPTION
**Pull request by babaknaderi**
fixing #33

### CODE REVIEW

File: ter [hunk 1]
```diff
@@ -0,0 +1,238 @@
+
+                   SSUUMMMMAARRYY OOFF LLEESSSS CCOOMMMMAANNDDSS
```
Comment (Artamus) [inline ter:2]: Why is this file here?

Comment (babaknaderi) [reply ter:2]: can you please remove it?

Comment (Artamus) [reply ter:2]: Sure thing :)
